includes/Renderer/Story/HTML.php

```diff
@@ -67,6 +68,7 @@ public function __construct( Story $story ) {
 	 */
 	public function render(): string {
 		$markup = $this->story->get_markup();
+		$markup = $this->fix_malformed_script_link_tags( $markup );
```

Comment:
Could we maybe only conditionally run this logic. Like checking for `VIP_GO_ENV` const. [See](https://github.com/Automattic/vip-go-mu-plugins/blob/291e8d413545ccc6a537e9415bc7e9ebce543e4e/000-vip-init.php#L83-L94).